pkg/reboot/calculator.go

```diff
@@ -78,6 +105,17 @@ func (s *safeTimeCalculator) calcMinTimeAssumeRebooted() {
 	minTime += 15 * time.Second
 	s.log.Info("calculated minTimeToAssumeNodeRebooted is:", "minTimeToAssumeNodeRebooted", minTime)
 	s.minTimeToAssumeNodeRebooted = minTime
+
+	if s.timeToAssumeNodeRebooted < minTime {
+		err := fmt.Errorf("snr agent can't start requested value for timeToAssumeNodeRebooted is too low")
```

Comment:
NIT: it took me a couple of seconds to understand the error message. Could you consider to use something to separate the problem from it's cause?
Something like:

"snr agent can't start: requested value for timeToAssumeNodeRebooted is too low"